messaging/kafka-avro-reactive-messaging/src/test/java/io/quarkus/ts/openshift/messaging/kafka/SchemaRegistryContainer.java

```diff
@@ -0,0 +1,41 @@
+package io.quarkus.ts.openshift.messaging.kafka;
+
+import io.strimzi.StrimziKafkaContainer;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+
+public class SchemaRegistryContainer extends GenericContainer<SchemaRegistryContainer> {
```

Comment:
I would move these classes to a separate package, so we can directly see what are the tests and what the resources/containers.